activity/server.js

```diff
@@ -16,19 +29,25 @@ const sequelize = new Sequelize('postgres', 'postgres', 'example', {
   },
 })
 
-
+// if we try to access 
 app.get('/', (req, res) => {
   sequelize
     .authenticate()
-      .then(() => {
-        res.send("Successful authentication")
-      })
-      .catch(err => {
-        console.error("Error: ", err)
-        res.send("faild to authenticate")
-      })
+    .then(() => {
+      res.send("successful authentication")
+    })
+    .catch(err => {
+      console.error("Error: ", err)
+      res.send("failed to authenticate")
+    })
 })
 
+// telling the app that localhost:8080/activity
+// will use the code in the activity.js file
+app.use("/activity", activity);
+
+// serving at port 8000, NOT TO BE CONFUSED WITH 8080,
```

Comment:
let's get rid of comments like this from master. Ok to have in your branches but not useful for master

Reply:
makes sense.
replaced with `// serving at port 8000`